tests/test_message_template.py

```
@@ -382,6 +383,82 @@ def test_session_expired_is_configurable(self):
+        "account_blocked",
+        "unauthorized_user",
+        "user_not_found",
+        "self_excluded",
+        "terms_not_accepted",
+        "terms_version_outdated",
+        "otp_attempts_exceeded",
+        "two_factor_inactive",
+        "betting_time_expired",
+        "session_expired",
+        "account_blocked_by_request",
+    ]
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)